src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,9 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+
+    public boolean isMutating() {
```

Comment:
This method should have a header comment. If not, users of the method as well as those who overrides method will not know how exactly the method behaves.

Reply:
Having a default implementation for the `isMutating` method helps to reduce the amount of code. However, some may argue that choosing `false` (or `true`) as the default is arbitrary and hurts the intuitiveness of the code. Making the method `abstract` forces all child classes to declare whether it is a mutating command or not, rather than depend on an arbitrary decision made in the parent class.